packages/ddp-client/common/namespace.js

```diff
@@ -89,3 +89,9 @@ DDP.onReconnect = callback => DDP._reconnectHook.register(callback);
 DDP._allSubscriptionsReady = () => allConnections.every(
   conn => Object.values(conn._subscriptions).every(sub => sub.ready)
 );
+
+// Sometimes data is loaded using a method, this hack provide a way
```

Comment:
Saying this is a hack is not good for an official function.
Please use JSDoc to document this function properly.